mater/lib/src/v2/reader.rs

```diff
@@ -1,12 +1,17 @@
-use ipld_core::cid::Cid;
-use tokio::io::{AsyncRead, AsyncReadExt, AsyncSeekExt, AsyncWriteExt, BufReader};
+use std::collections::HashSet;
 
 use super::index::read_index;
+use crate::multicodec::DAG_PB_CODE;
 use crate::{
     v2::{index::Index, Characteristics, Header, PRAGMA},
     Error,
 };
-
+use ipld_core::cid::Cid;
+use ipld_core::codec::Codec;
```

Comment:
Looks like the formatting will break on this one (import order): `cargo +nightly fmt` I guess?